api/__init__.py

```diff
@@ -8,10 +8,13 @@
 from flask_smorest import Api
 from flask_sqlalchemy import SQLAlchemy
 
+from api.utils import ToolhubClient
+
 api = Api()
 db = SQLAlchemy()
 migrate = Migrate()
 oauth = OAuth()
+thc = ToolhubClient()
```

Comment:
A more explicit name would make it easier to understand what this is when encountered in other parts of the code, e.g. 'toolhub_client'